### PR TITLE
BigQuery Log Alerting Fixes

### PR DESCRIPTION
**Pull request by daniel-cit**
The changes for module BigQuery Log Alerting  in this PR are:

- fixes in the documentation to make the requirements more clear specially regarding the use of a service account to deploy the module.
- the variable `region` has been split in two new variables `bigquery_location` and `function_region`.

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: examples/bq-log-alerting/versions.tf [hunk 1]
```diff
@@ -15,5 +15,9 @@
  */
 
 terraform {
-  required_version = ">=0.12.6, <0.14"
+  required_version = ">=0.12.6"
```
Comment (aaronsutton) [inline examples/bq-log-alerting/versions.tf:18]: In the README it indicates that Terraform 13+ is required. modules/bq-log-alerting/README.md:162

Comment (daniel-cit) [reply examples/bq-log-alerting/versions.tf]: @aaronsutton updated to `0.13.6`

Comment (daniel-cit) [reply examples/bq-log-alerting/versions.tf:18]: It looks like we can't use a newer version
```
The configuration is valid.
terraform_validate ./examples/bq-log-alerting 
Error: 
Unsupported Terraform Core version
 on versions.tf line 18, in terraform:
 18:   
required_version = ">=0.13.6"
This configuration does not support Terraform version 0.13.5. To proceed,
either choose another supported Terraform version or update this version
constraint. Version constraints are normally set for good reason, so updating
the constraint may lead to other errors or unexpected behavior.
```

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**